components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -1522,6 +1532,23 @@ public static AuthenticationContext getAuthenticationContextFromCache(String con
         return authenticationContext;
     }
 
+    public static Map<String, String> getAuthenticatorPropertyMapForLocalAuthenticators(
+            String name, String tenantDomain) {
+
+        Map<String, String> propertyMap = new HashMap<>();
+        try {
+            LocalAuthenticatorConfig authenticatorConfig = ApplicationAuthenticatorService.getInstance()
+                    .getUserDefinedLocalAuthenticator(name, tenantDomain);
+            for (Property property : authenticatorConfig.getProperties()) {
+                propertyMap.put(property.getName(), property.getValue());
+            }
```

Comment:
This seems to load only Userdefined authenticators

Is the function name correct and intended ? if so I think logic should change. Else function name should denote this is about user defined authenticators

Reply:
Will change the name. For system local authenticators, we do not configure any properties, therefore no need to do this logic for them.